lambdas/validator.py

```diff
@@ -74,13 +73,21 @@ def parse_payload(event: dict) -> InputPayload:
     body = json.loads(event["body"]) if "requestContext" in event else event
 
     try:
-        return InputPayload(**body)
+        return InputPayload(
+            **body,
+        )
     except Exception as exc:
         message = f"Invalid input payload: {exc}"
         logger.error(message)  # noqa: TRY400
         raise ValueError(message) from exc
 
 
+def validate_secret(challenge_secret: str | None) -> None:
```

Comment:
Commenting here but this comment applies to the "Lambda helper functions" (i.e., all the functions in this module excluding `lambda_handler`): I like the general pattern of the module with the Lambda handler containing _only_ the Lambda handler and storing all the helper functions in its own module (within `lambdas/`). Just a thought! 🤔

Reply:
My pushback would be that all the functions this file are 100% related towards fulfilling a lambda invocation: parsing payloads, validating challenges, preparing response payloads, and the ping/validate handlers.  

The real complexity is pushed down into `aip.py`.  

I take your point, but my preference would be to keep it here.  That said, if it gets much more (at all?) more complicated, I think I'd agree.  This feels like an inflection point where it's still simple enough to keep here.